wiki/Registration/en.md

```diff
@@ -1,26 +1,21 @@
----
-outdated: true
----
-
 # Registration
 
 *Warning: Having more than one osu! user account at any time is an infringement against the [osu! rules](/wiki/Rules)!*
 
-1. After [installing](/wiki/Installation) osu! and running it, osu! will prompt you to sign in or register. Assuming that you're here because you need an account, click `Create an account` and keep osu! opened. This should open the [Create Account](https://osu.ppy.sh/p/register) web page.
+1. After [installing](/wiki/Installation) osu! and running it, osu! will prompt you to sign in or register. Assuming that you're here because you need an account, click `Create an account`. Enter your username (if it's not taken), your email, and your password. Then click `Create Account`.
```

Comment:
```suggestion
1. After [installing](/wiki/Installation) osu! and running it, osu! will prompt you to sign in or register. Assuming that you're here because you need an account, click `Create an account`.
```